error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/ImmutableMultisetRulesTestInput.java

```diff
@@ -26,6 +26,7 @@ ImmutableMultiset<ImmutableMultiset<Integer>> testEmptyImmutableMultiset() {
         Stream.<Integer>empty().collect(toImmutableMultiset()));
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
Here and below: these annotations are now required, as the files are now compiled.